frontend/add-client/form-inputs/contact-information-inputs.component.tsx

```diff
@@ -160,8 +160,12 @@ export default React.forwardRef(function ContactInformationInputs(
 });
 
 function getTodayAsString() {
-  const date = new Date().toISOString();
-  return date.slice(0, date.indexOf("T"));
+  const day = new Date().getDate();
+  const year = new Date().getFullYear();
+  const month = new Date().getMonth();
+
+  const date = `${year}-${month > 10 ? "" : "0"}${month + 1}-${day}`;
```

Comment:
Is this fixing a bug with time zones or something? Or why the change here?

Reply:
I think it may be related, or at least a similar bug. After 5 it was showing as the next day.

Reply:
I'd rather use https://github.com/felixge/node-dateformat to format the dates correctly, and also find a solution that we apply to all places where we're using dateformat.

Maybe try fixing it with dateformat?

Reply:
`dateFormat(new Date(), "yyyy-mm-dd")`

Reply:
resolved in 96-02 fixes